requirements/dev.txt

```diff
@@ -118,7 +118,7 @@ django==3.2.11
     #   edx-toggles
     #   event-tracking
     #   jsonfield
-django-config-models==2.2.2
+django-config-models==2.3.0
```

Comment:
Only CI changes in new version https://github.com/openedx/django-config-models/blob/master/CHANGELOG.rst#230---2022-01-19